spec/teacher_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'date'
+require_relative '../person/teacher'
+
+describe Teacher do
```

Comment:
- [ ] Greetings! 🌟 It's important to bolster test coverage for all methods present in the Teacher class, especially the to_s method, which is currently untested. By addressing this, you will be in perfect alignment with the project's requirements. 🧪 This step is crucial to validate the effectiveness and completeness of your code. 🚀